tests/MessageTest.php

```diff
@@ -1057,4 +1057,14 @@ public function testGbkCharsetDecoding(): void
 
         static::assertSame('Hi', \trim($message->getDecodedContent()));
     }
+    
+    
+    public function testHtmlMultipartMessage(): void
+    {
+        $this->mailbox->addMessage($this->getFixture('HTMLmultipartMessage'));
+
+        $message = $this->mailbox->getMessage(1);
+        
```

Comment:
This is a good start. What it's missing here are the expectation written in form of assertions, for example:

```php
static::assertSame('<html>', $message->getBodyHtml());
```
What should you assert? What you expect, so if you told in the issue `only returns the first part of a multipart/mixed message` test the returned content in a way you describe your expectations.